src/components/Note/Note.tsx

```diff
@@ -12,6 +12,9 @@ import {Actions} from "store/action";
 import {Participant} from "types/participant";
 import "./Note.scss";
 import {getEmptyImage} from "react-dnd-html5-backend";
+import {addProtocol} from "utils/images";
+import {getColorClassName} from "constants/colors";
```

Comment:
Can be removed because the columns accent color is already provided by the column component. Since notes are children of columns, you can use the accent colors introduced by the column.